src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -8156,11 +8184,11 @@ namespace OutputReportTabular {
 
             //---- End Use Sub-Table
             rowHead.allocate(16);
-            columnHead.allocate(6);
-            columnWidth.allocate(6);
-            columnWidth = 14; // array assignment - same for all columns
-            tableBody.allocate(6, 16);
-            for (iResource = 1; iResource <= 6; ++iResource) {
+            columnHead.allocate(13);
+            columnWidth.allocate(13);
+            columnWidth = 10; // array assignment - same for all columns
+            tableBody.allocate(13, 16);
+            for (iResource = 1; iResource <= 13; ++iResource) {
```

Comment:
But it's 13 here? Looks like 13 is correct, but this needs to be double-checked.

Reply:
It is 13 here because this FOR loop counts water.